frontend/src/views/v-zoom.vue

```diff
@@ -94,6 +94,9 @@
       v-bind:key="slice.hash",
       v-bind:class="{ 'message-body active': slice.messageBody !== '' }"
     )
+      .tooltip
+        | Commit hash: {{ slice.hash.substr(0, 4) }} &nbsp;
```

Comment:
I am actually not sure whether 4 digits is enough for identifying a commit in a repository, especially for project with a large number of commits. Is it possible to change it to 7 digits instead?

Reply:
Yes, I too think 7 is better.